### DISCUSSION TITLE
Write for us

### DISCUSSION THREAD

**Discussion by aeliavision** [Deadshot0x7/007-TheBond#3] [General]
I'm sorry for writing here but it was the only way to contact you,
You are good in the cybersecurity field? Then you can write for us about your own tool on GitHub and the use of it.
Your article will be published with your name and a link to your website and social media accounts if you want. Please reply to us at Email: kalitutwrite @ gmail.com
Walid Salame
kalitut .com
Twitter:@xKaliSec

**Comment by Deadshot0x7** [comment]
Hey, I'm interested to write a blog on your website, But I'm a bit busy right now, I will contact you soon

**Comment by aeliavision** [comment]
OK :)

**Comment by Deadshot0x7** [comment]
Hey @thexwts , I have some queries regarding the blog and the query is that does I will able to update them?
